apps/designer/app/designer/features/style-panel/sections/spacing/value-text.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+import { parseCssValue } from "../../shared/parse-css-value";
+import { SpacingLayout } from "./layout";
+import { ValueText as ValueTextComponent } from "./value-text";
+
+const Template = (
+  args: Pick<
+    React.ComponentProps<typeof ValueTextComponent>,
+    "source" | "value"
+  >
+) => {
+  const [hovered, setHovered] = React.useState<{ property: string }>();
+  return (
+    <SpacingLayout
+      onHover={setHovered}
+      onClick={() => null}
+      renderCell={({ property }) => (
+        <ValueTextComponent
+          isActive={property === hovered?.property}
+          {...args}
+        />
+      )}
+    />
+  );
+};
+
+export default {
+  title: "Spacing/ValueText",
+  component: Template,
+  argTypes: {
+    source: { control: "select", options: ["set", "unset", "inherited"] },
+    value: {
+      control: "select",
+      options: Object.fromEntries(
+        [
+          "0",
+          "10rem",
+          "100rem",
+          "100px",
+          "1000px",
+          "10000px",
+          "100000px",
+          "auto",
+          "revert-layer",
+        ].map((value) => [value, parseCssValue("marginTop", value)])
+      ),
+    },
+  },
+} as ComponentMeta<typeof Template>;
+
+export const ValueText: ComponentStory<typeof Template> = Template.bind({});
```

Comment:
do you need to use the Template thing? storybook's latest format uses plain component export, e.g. combobox.stories

Reply:
I use Template to control `args` (to make it possible to change some of the props via Storybook UI and see how component behaves)

Reply:
you can put args on the component directly, see text-editor.stories for example

Reply:
Template also changes render a lot. Take a look at what it does